runtime/lua/vim/lsp/util.lua

```diff
@@ -674,7 +674,10 @@ function M.focusable_float(unique_name, fn)
       return
     end
   end
-  local pbufnr, pwinnr = fn()
+  local ok, pbufnr, pwinnr = pcall(fn())
```

Comment:
I also get this error, but maybe we should find which function returns an invalid pwinnr ?
I'm going to investigate, and maybe provide a better solution for this (or maybe this is the solution but imo it just avoids the real issue).

Reply:
indeed the function is called for each hit in insert mode move which should not be happening. I think this might be related to [completion nvim](https://github.com/nvim-lua/completion-nvim/)

Reply:
it has been fixed in lastest master, just update the plugin !

Reply:
awesome!  Does this mean that the commit is useless or should we "protect" the function call?  Its quite cumbersome when things are not working.

Side note, a better logging system in general would be nice for this kind of work.  It would be really nice to know that your function is throwing, but not display red messages to the end user.

Reply:
```suggestion
  local ok, pbufnr, pwinnr = pcall(fn)
```